packages/snaps-utils/src/handlers.ts

```diff
@@ -34,7 +35,7 @@ export type OnRpcRequestHandler = (args: {
  * If the snap has no insights about the transaction, this should be `null`.
  */
 export type OnTransactionResponse = {
-  insights: { [key: string]: Json } | null;
+  ui: Component;
```

Comment:
We're (or will be) using `content` rather than `ui` in `snap_dialog`.

```suggestion
  content: Component;
```

Reply:
👍